packages/design-picker/src/components/style-variation-badges/badge.tsx

```diff
@@ -18,7 +21,21 @@ const Badge: React.FC< BadgeProps > = ( { variation } ) => {
 	}
 
 	return (
-		<div className="style-variation__badge-wrapper">
+		<div
+			className="style-variation__badge-wrapper"
+			tabIndex={ 0 }
+			role="button"
```

Comment:
We need to add `aria-label` for the a11y

Reply:
Since we don't have the style variation title here, I've only added "Preview with this style" as `aria-label`. @fushar we might need to pass the style title to the `v2/starter-designs` endpoint if we want to show it in the style indicator.

Reply:
Okay, we can return the title, I can work on it tomorrow 😄

Reply:
D87607-code